tests/ballerina-test/src/test/resources/test-src/structs/initializers/init.bal

```diff
@@ -0,0 +1,36 @@
+package initializers;
+
+public struct employee{
+    int age = 20;
+    string name;
+    string address;
+}
+
+public function <employee p> employee() {
+    p.age = 10;
+    p.name = "James";
+}
```

Comment:
Do we support initializing ref-types in the initializer? eg: a nested struct.. 
Shall we add a test case? :)

Reply:
Struct initializer can be detected during the compile time. I've created an issue to track this.